### ISSUE TITLE
Generate sitemaps with Controller and Action

### ISSUE THREAD

**Issue by bensheldon** [bensheldon/open311status#265]
Instead of using the S3 Sitemap generator, we should generate sitemaps via index and controllers and XML: https://www.johnnunemaker.com/rails-easy-sitemaps/

- At the leaf, each city by day (rather than month) should be its own Sitemap, which lists out individual requests for that city on that day (service requests are the main "content")
- There should be an index for each city that enumerates all of the previous days
- There should be a primary sitemap index that references each city index
- There can also be an additional "static URL sitemap" referenced from the root sitemap index, that can contain other more singular site links.

So in summary, a primary index references a per-city index, and the per-city index lists out a sitemap per-city-per-day that contains all of the service requests for that day.
